### DISCUSSION TITLE
blog/programmers-are-scientists

### DISCUSSION THREAD

**Discussion by giscus** [Akarikev/elorm#1] [General]
# blog/programmers-are-scientists

Exploring how programmers embody scientific principles in problem-solving and innovation.

http://localhost:3000/blog/programmers-are-scientists

**Comment by Akarikev** [comment]
test test

**Reply by Akarikev** [reply]
another test

**Reply by Akarikev** [reply]
test

**Comment by TreasureUzoma** [comment]
<b>Nice Article</b>
<p><i>Programmers</i> are indeed scientist</p>

**Comment by Dubby-Bantin** [comment]
this is really nice bro...programmers are scientists indeed

**Reply by Akarikev** [reply]
thanks for reading!

**Comment by champ3oy** [comment]
🔥🔥

**Reply by Akarikev** [reply]
🔥🔥
